images/async-gateway/Dockerfile

```diff
@@ -1,16 +1,21 @@
 FROM golang:1.15 as builder
 
-COPY async-gateway/go.mod async-gateway/go.sum /workspace/async-gateway/
-WORKDIR /workspace/async-gateway/
+COPY go.mod go.sum /go/src/github.com/cortexlabs/cortex/
+WORKDIR /go/src/github.com/cortexlabs/cortex
```

Comment:
revert to the simpler directory structure. we are using go modules, there's no need to use a GOPATH structure, it just makes it harder to read and modify

Reply:
I buy this. I suppose we should do the same to the operator as well. The operator was used as a blueprint.